enhancements/operator-sdk/downstream-operator-sdk.md

```diff
@@ -0,0 +1,366 @@
+# find . -mindepth 2 -name OWNERS -exec git rm -f '{}' +
+
+# update vendor
+go mod vendor
+git add vendor
+
+# preserve our version of these files
+git checkout HEAD -- OWNERS Makefile .gitignore
+
+# unmerged files are overwritten with the upstream copy
+unmerged_files=$(git diff --name-only --diff-filter=U --exit-code)
+differences=$?
+if [[ $differences -eq 1 ]]; then
+  unmerged_files_oneline=$(echo "$unmerged_files" | paste -s -d ' ')
+  git checkout --theirs -- $unmerged_files_oneline
+  if [[ $(git diff --check) ]]; then
+    echo "All conflict markers should have been taken care of, aborting."
+    exit 1
+  fi
+  git add -- $unmerged_files_oneline
+else
+  unmerged_files="<NONE>"
+fi
+
+# update upstream Makefile changes, but don't overwrite build patch
+# script executor must manually decline the build patch change, the diff may
+#  contain other changes that need accepting
+git show tags/"$version":Makefile > Makefile.sc
+git add --patch --interactive Makefile.sc
+git checkout Makefile.sc
+
+# bump UPSTREAM-VERSION file
+echo "$version" > UPSTREAM-VERSION
+git add UPSTREAM-VERSION
+
+# just to make sure an old version merge is not being made
+git diff --staged --quiet && { echo "No changed files in merge?! Aborting."; exit 1; }
+
+# make local commit
+git commit -m "Merge upstream tag $version" -m "Merge executed via ./UPSTREAM-MERGE.sh $version $upstream_remote $rebase_branch" -m "$(printf "Overwritten conflicts:\\n%s" "$unmerged_files")"
+
+# verify merge is correct
+git --no-pager log --oneline "$(git merge-base origin/"$rebase_branch" tags/"$version")"..tags/"$version"
+
+printf "\\n** Upstream merge complete! **\\n"
+echo "View the above incoming commits to verify all is well"
+echo "(mirrors the commit listing the PR will show)"
+echo ""
+echo "Now make a pull request, after it's LGTMed make the tag:"
+echo "$ git checkout $rebase_branch
+$ git pull
+$ git tag <origin version>-$version
+$ git push origin <origin version>-$version"
+```
+
+#### Handling Patches
+
+Typically we want to submit all bugs upstream, have them release then sync the
+release downstream. But there will be times we need to fix a problem downstream
+to resolve a bug that we can’t wait for an upstream release. The process here
+will be twofold.
+
+1. For fixes to vendored dependencies, have a patch in the patches directory
+   that will be applied to the vendor after we run go mod vendor.
+1. For fixes to mainline code, fix the mainline code directly. NOTE: these changes
+   will be lost when re-running UPSTREAM-MERGE.sh script.
+
+For either of the above, we will always submit the patches to the appropriate
+upstream repos.
+
+#### Directory Structures
+
+The following is a list of directories and files that we will add to the repos:
+
+* UPSTREAM-MERGE.sh # script used to sync the upstream to the downstream
+* ci
+  * dockerfiles
+    * variety of Dockerfiles used to build images for running tests
```

Comment:
These images also cames from the legacy. See https://github.com/operator-framework/operator-sdk/tree/v0.11.x/ci/dockerfiles. It no longer exists in 1.0.0. have we any reason to keep that? Why do we need to build these images?